ci_framework/roles/devscripts/README.md

```diff
@@ -0,0 +1,235 @@
+* `cifmw_devscripts_crb_repo` (str) Repo URL of code ready builder.
+* `cifmw_devscripts_dns_entries` (dict) having `api` & `ingress` keys whose value hold the clusters
+  api & ingress IP addresses. Required when using local resolution.
+* `cifmw_devscripts_dry_run` (bool) If enabled, the workflow is evaluated.
+* `cifmw_devscripts_extra_net_infra` (list) A list of extra networks to be created. Networks is a key/value
+  pairs. Refer [section](#supported-keys-in-cifmw_devscripts_extra_net_infra) for more information
+* `cifmw_devscripts_make_target` (str) Optional, the target to be used with dev-scripts.
+* `cifmw_devscripts_ocp_version` (str) The version of OpenShift to be deployed.
+* `cifmw_devscripts_osp_compute_nodes` (list) A list of nodes which has key/value pairs
+  containing details about OpenStack compute nodes. Refer
+  [section](#supported-keys-in-cifmw_devscripts_osp_compute_nodes) for more information.
+* `cifmw_devscripts_pull_secret` (str) Access secret for pulling OCP component images.
+* `cifmw_devscripts_src_dir` (str) The parent folder of dev-scripts repository.
+
+### Supported keys in cifmw_devscripts_config_overrides
+
+| Key | Default Value | Description |
+| --- | ------------- | ----------- |
+| working_dir | `/home/dev-scripts` | Path to the directory to store script artifacts. |
+| openshift_version | | The version of OpenShift to be deployed. |
+| openshift_release_type | | Type of OpenShift release. Supported values are `nightly\|ga\|okd` |
+
+#### General settings
+
+| Key | Default Value | Description |
+| --- | ------------- | ----------- |
+| cluster_name | `ocp` | Name for the ocp cluster. |
+| base_domain | `openstack.lab` | Base domain to be used for the cluster. |
+| ssh_pub_key | | SSH public key to enable access to the nodes part of OCP cluster. |
+| ntp_servers | `clock.corp.redhat.com` | NTP servers to be configured in the cluster. |
+
+#### OpenShift networking
+
+| Key | Default Value | Description |
+| --- | ------------- | ----------- |
+| ip_stack | `v4` | IP stack for the cluster. Supported values are `v4\|v6\|v6v4`. |
+| network_type | `OpenShiftSDN` | Sets the network type for the OpenShift cluster. Supported values are `OpenShiftSDN\|OVNKubernetes`. |
+| provisioning_network_profile | `Disabled` | Allow the script to manage the provisioning network. Supported values are `Disabled\|Managed`. |
+| manage_pro_bridge | `n` | Allow the script to manage the provisioning bridge. Supported values are `y\|n`. |
+| provisioning_network | | The subnet CIDR to be used for the provisioning network. |
+| pro_if | | The network interface to be attached to the provisioning bridge. |
+| manage_int_bridge | `n` | Allow the script to manage the internal bridge. Supported values are `y\n`. |
+| int_if | | The network interface to be attached to the internal cluster bridge. |
+| manage_br_bridge | `n` | All the script to manage the external bridge. Supported values are `y\|n`. |
```

Comment:
`s/All/Allow`

Reply:
Thank you.
I have made the changes in the new commit.